spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,158 @@
+require "rails_helper"
+require_relative "form/helpers"
+
+RSpec.describe "Home Page Features" do
+  include Helpers
+
+  context "when the user is a data provider" do
+    let(:user) { FactoryBot.create(:user, name: "Provider") }
+
+    before do
+      create_list(:lettings_log, 6, :in_progress, owning_organisation: user.organisation, created_by: user)
+      create_list(:lettings_log, 2, :in_progress, owning_organisation: user.organisation)
+      create_list(:lettings_log, 4, :completed, owning_organisation: user.organisation, created_by: user)
+      create_list(:lettings_log, 2, :completed)
+      sign_in user
+      visit("/")
+    end
+
+    it "displays the correct welcome text" do
+      expect(page).to have_current_path("/")
+      expect(page).to have_content("Welcome back, Provider")
+      expect(page).to have_content("Complete your logs")
+    end
+
+    context "when their organisation has submitted sales logs" do
+      before do
+        create_list(:sales_log, 5, :in_progress, owning_organisation: user.organisation, created_by: user)
+        create_list(:sales_log, 3, :completed, owning_organisation: user.organisation, created_by: user)
+        visit("/")
+      end
+
+      it "displays correct data boxes, counts and links" do
+        data_boxes = page.find_all(class: "app-data-box-one-half")
+        expect(data_boxes.count).to eq(2)
+        expect(data_boxes[0].all("a").map(&:text)).to eq(["6", "Your lettings in progress", "View all lettings"])
+        expect(data_boxes[0].all("a").map { |line| line["href"] }).to eq([lettings_logs_path(status: [:in_progress], assigned_to: "you"), lettings_logs_path(status: [:in_progress], assigned_to: "you"), clear_filters_path(filter_type: "lettings_logs")])
+        expect(data_boxes[1].all("a").map(&:text)).to eq(["5", "Your sales in progress", "View all sales"])
+        expect(data_boxes[1].all("a").map { |line| line["href"] }).to eq([sales_logs_path(status: [:in_progress], assigned_to: "you"), sales_logs_path(status: [:in_progress], assigned_to: "you"), clear_filters_path(filter_type: "sales_logs")])
+      end
+    end
```

Comment:
I wonder if there's a different way to test this so that we're not locked into the dom as much, but if we're not expecting it to change soon or much, I'm happy with this